src/less/components/tables.less

```diff
@@ -72,7 +81,7 @@
     // Plain table
     &.table-plain {
         tbody {
-            > tr:not(:last-child) {
```

Comment:
Double check why this is removed

Reply:
This was removed because last-child should also have a border-bottom